src/utils.c

```diff
@@ -21,6 +23,8 @@
 #endif
 #endif
 
+const int MAX_PATH_LEN = 1024;
```

Comment:
Use macro or `enum` instead. In C, an object with const integer type occupies extra memory space.

Reply:
Learned and changed.